src/eradiate/scenes/atmosphere/_homogeneous.py

```diff
@@ -142,80 +143,26 @@ def phase(self) -> PhaseFunction:
     def eval_mfp(self, ctx: KernelDictContext) -> pint.Quantity:
         # Inherit docstring
         return (
-            1.0 / self.eval_sigma_s(ctx.spectral_ctx)
-            if self.eval_sigma_s(ctx.spectral_ctx).m != 0.0
-            else 1.0 / self.eval_sigma_a(ctx.spectral_ctx)
+            1.0 / self.eval_sigma_s(ctx.si)
+            if self.eval_sigma_s(ctx.si).m != 0.0
+            else 1.0 / self.eval_sigma_a(ctx.si)
         )
 
-    def eval_albedo(self, spectral_ctx: SpectralContext) -> pint.Quantity:
-        """
-        Return albedo.
+    def eval_albedo(self, si: SpectralIndex) -> pint.Quantity:
+        """Return albedo at given spectral index."""
+        return self.eval_sigma_s(si) / (self.eval_sigma_s(si) + self.eval_sigma_a(si))
 
-        Parameters
-        ----------
-        spectral_ctx : .SpectralContext
-            A spectral context data structure containing relevant spectral
-            parameters (*e.g.* wavelength in monochromatic mode).
```

Comment:
Deleting all docstrings isn't a good idea IMO: the information about the returned type, in particular, is very useful.

Reply:
I found these docstrings somehow redundant, they all over the place and they are exactly the same. I thought there was a way to write the docstrings once (in `Atmosphere`) and then inherit docstrings in subclasses, isn't that possible?
Otherwise, I'll gladly restore the docstrings, all that was still work in progress

Reply:
These methods haven't made it to the `Atmosphere` interface yet, have they?

Reply:
No, you are right.
They are in `AbstractHeterogeneousAtmosphere` and `HomogeneousAtmosphere`, the difference (as far as I can see) between the `eval_*` methods in these two is the presence of the `zgrid` parameter.